### PR TITLE
Added Reminders API

### PR DESCRIPTION
**Pull request by omenocal**
Added methods for Alexa's Reminders API

Official documentation reference:
https://developer.amazon.com/docs/smapi/alexa-reminders-api-reference.html

### CODE REVIEW

File: docs/alexa-apis.rst [hunk 1]
```diff
@@ -540,3 +540,216 @@ To send a card requesting user the permission to read/write Alexa lists, you can
...
+      .setCreatedTime("2018-12-11T14:05:38.811")
+      .setTriggerAbsolute("2018-12-12T12:00:00.000")
+      .setTimeZoneId("America/Denver")
+      .setRecurrenceFreqDaily()
+      .addContent("en-US", "CREATION REMINDER TEST")
+      .enablePushNotification();
+
+    const reminderResponse = await voxaEvent.alexa.reminders.createReminder(reminder);
+
+    voxaEvent.model.reminder = reminderResponse;
```
Comment (armonge) [inline docs/alexa-apis.rst]: why do we need to add it to the model? @omenocal

Comment (omenocal) [reply docs/alexa-apis.rst]: It's not necessary, it's just an example on how to display the result from the API to a view

File: src/platforms/alexa/apis/Reminders.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    return this;
+  }
+
+  public disablePushNotification(): ReminderBuilder {
+    this.pushNotificationStatus = "DISABLED";
+
+    return this;
+  }
+
+  public build(): any {
```
Comment (armonge) [inline src/platforms/alexa/apis/Reminders.ts]: isn't there a type for this?

File: test/alexa/Reminders.spec.ts [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+import { views } from "./../views";
+
+const reqheaders = {
+  accept: "application/json",
+  authorization: "Bearer apiAccessToken",
+  host: "api.amazonalexa.com",
+};
+
+describe("Reminders", () => {
+  let event: any;
```
Comment (armonge) [inline test/alexa/Reminders.spec.ts]: what's this type?

**Pull request merged**